app/code/Magento/Ui/view/base/web/js/grid/columns/image-preview.js

```diff
@@ -174,6 +193,27 @@ define([
                 block: 'center',
                 inline: 'nearest'
             });
-        }
+        },
+
+        /**
+         * Set image preview keyboard navigation listener
+         */
+        setNavigationListener: function () {
+            var key,
+                self = this,
+                adobeModalSelector = '.adobe-stock-modal';
+
+            $(document).on('keydown', function (e) {
+                key = keyCodes[e.keyCode];
+
+                if ($(adobeModalSelector).hasClass('_show') && $(self.previewImageSelector).length > 0) {
```

Comment:
It's better to unbind this check from visual representation as well
```suggestion
                if (this.visibleRecord() !== null) {
```

Reply:
Fixed this, thanks!